test/e2e-test/pages/eligibility-questions/eligibility-question-page.ts

```diff
@@ -15,7 +15,7 @@ module.exports = {
     });
 
     When('I go to the second eligibility question without answering the first', async () => {
-      I.amOnPage(testUrl + paths.eligibility.questions + '?id=1');
+      I.amOnPage(testUrl + paths.common.questions + '?id=1');
```

Comment:
`paths.common.questions` should we place this under `paths.common.eligibility.questions`

Reply:
I am just leaving the paths object as simple as possible, we don't need 2-3 object levels as it doesn't bring any benefit or any more information IMO, so paths object is just a 2 levels deep object, the first level is for appeal state, and inside that all the routes available for that state. Also, that saves me time and complexity so don't need to flatten the paths object in the middleware